mod/deckorate_core/set/type/metric/export.rb

```diff
@@ -127,12 +127,12 @@ def metric_jsonld(a)
       "title" => a[:title],
       "designer" => "#{request&.base_url}/#{a[:designer]}",
       "question" => a[:headquarters],
-
       "metric_type" => a[:metric_type],
       "research_policy" => a[:research_policy],
       "about" => sanitize_html(a[:about]),
       "methodology" => sanitize_html(a[:methodology]),
       "topics" => get_topics(a[:topics]),
+      "framework_mappings" => get_topics(a[:topic_frameworks]),
```

Comment:
This is causing errors in the tests. I think the `#get_topics` method is expecting a topic, not a framework